resources for AL Home (#7474)
microsoft/AL: News.md

# 2023-08-16 Get AL VSIX preview versions on Visual Studio Code Marketplace

> AL VSIX prerelease versions are now available through the existing Visual Studio Code mechanism for getting notified about and installing prerelease versions. To learn more about this mechanism, visit [Publishing Extensions](https://code.visualstudio.com/api/working-with-extensions/publishing-extension#prerelease-extensions) in the Visual Studio Code documentation.

Reviewer: ```suggestion
AL Language extension Pre-Release versions are now available through the existing Visual Studio Code mechanism for getting notified about and installing Pre-Release versions. To learn more about this mechanism, visit [Publishing Extensions](https://code.visualstudio.com/api/working-with-extensions/publishing-extension#prerelease-extensions) in the Visual Studio Code documentation.
```

> To install the preview version, use the **Switch to Pre-Release Version** option in Visual Studio Code extension management page for the AL Language extension. **Note** that you can't install the prerelease directly from the Visual Studio Code Marketplace, you must install the public version first.

Reviewer: ```suggestion
To install the Pre-Release Version, use the **Switch to Pre-Release Version** option in Visual Studio Code extension management page for the AL Language extension. **Note** that you can't install the Pre-Release directly from the Visual Studio Code Marketplace, you must install the Release Version first.
```

Reviewer: ![image](https://github.com/microsoft/AL/assets/9417259/e7b13c3b-e233-48a6-b780-0d258f7f6fe0)

Reviewer: Here's ours
![image](https://github.com/microsoft/AL/assets/9417259/4376b8fb-e1fa-4629-8002-a1d7e5576975)

> ![Switch to prerelease version](https://raw.githubusercontent.com/microsoft/AL/master/resources/al_release.png)

Reviewer: ```suggestion
![Switch to Pre-Release Version](https://raw.githubusercontent.com/microsoft/AL/master/resources/al_release.png)
```
